README.md

```diff
@@ -285,26 +275,10 @@ The demo.html file should import the bundle.js and use the custom element:
 Note that the demo files should have the names demo.js and demo.html for the
 JavaScript and html files respectively.
 
-We have the definition, the demo, the entry file. All that is left in order to
-build the component is to specify which files will be added to the npm package.
-This is done in the package.json file using the files field:
 
-```
-  "files": [
-    "LICENSE",
-    "README.md",
-    "demo.html",
-    "index.js",
-    "script.js",
-    "cjs/",
-    "umd/"
-  ],
-```
-*Skip the files property if you want to include all files.*
-
-Use the LICENSE template.
+Make sure all files have the LICENSE notice at the top.

```

Comment:
You can add here a few words how to do it automatically with the script that adds copyrights

Reply:
Added "Run `npm run add:copyright` to automatically add copyright notice to all files."